testsuite/features/step_definitions/common_steps.rb

```diff
@@ -69,21 +69,24 @@
 Then(/^the uptime for "([^"]*)" should be correct$/) do |host|
   node = get_target(host)
   uptime_seconds, _return_code = node.run("awk '{print $1}' /proc/uptime") # run code on node only once, to get uptime in seconds
-  uptime_days = (uptime_seconds / 86400.0) # 60 seconds * 60 minutes * 24 hours; the .0 forces a float division
-  uptime_hours = (uptime_seconds / 3600.0) # 60 seconds * 60 minutes
-  uptime_minutes = (uptime_seconds / 60.0) # 60 seconds
+  uptime_minutes = (uptime_seconds.to_f / 60.0) # 60 seconds; the .0 forces a float division
+  uptime_hours = (uptime_minutes / 60.0) # 60 minutes
+  uptime_days = (uptime_hours / 24.0) # 24 hours
 
   # rounded values to nearest integer number
   rounded_uptime_minutes = uptime_minutes.round
   rounded_uptime_hours = uptime_hours.round
-  rounded_uptime_days = uptime_days.round
+
+  # needed for the library's conversion of 24h multiples plus 11 hours to consider the next day
+  eleven_hours_in_seconds = 11 * 3600
```

Comment:
What about set directly 39600 and in a comment we write 11 * 3600 ? :) One less calculation.